docs/source/app-dev/command-deduplication.rst

```diff
@@ -90,7 +90,7 @@ Independently of how the outcome is communicated, command deduplication generate
 
 For deduplication to work as intended, all submissions for the same ledger change must be submitted via the same participant.
 Whether a submission is considered a duplicate is determined by completion events, and by default a participant outputs only the completion events for submissions that were requested via the very same participant.
-At this time, only `Daml driver for VMware Blockchain <https://www.digitalasset.com/daml-for-vmware-blockchain/>`__ supports command deduplication across participants.
+At this time, only `Daml driver for VMware Blockchain <https://docs.vmware.com/en/VMware-Blockchain/index.html>`__ supports command deduplication across participants.
```

Comment:
@digital-asset/daml-product  Do we want to link to VMWare site here?

Reply:
This page is specifically Daml related and mentions Daml.

Reply:
there's nothing in our docs that I can see

Reply:
Still, this feels like a product decision to me.

Reply:
let's just remove the link then..

Reply:
I've removed it.

Reply:
That's not a great outcome.

@stefanobaghino-da @bame-da can we get a product opinion on this please?

Reply:
The line can be removed entirely.